### ISSUE TITLE
Saving Output Image in the Python

### ISSUE THREAD

**Issue by ishansan** [zhreshold/mxnet-yolo#17]
Hi. I am pretty new to YOLO implementation. I was wondering which function I could call to save the output image/video to a directory?

The output image would contain the detected boundaries.

**Comment by zhreshold** [comment]
use matplotlib savefig function

**Comment by ishansan** [comment]
Yes @zhreshold . But how to do we retrieve the image with predicted boundaries before saving?
